src/containers/Subscription.js

```diff
@@ -1,26 +1,67 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
 import { APP_INFO, SUBSCRIPTION_FORM } from '../constants';
 import '../components/Auth/AuthModal.css';
 import Banner from '../components/UI/Banner';
 import { useKeycloak } from '@react-keycloak/web';
 import { useConfig } from '../hooks/useConfig';
+import axios from 'axios';
 
 export const Subscription = () => {
   const { keycloak } = useKeycloak();
   const config = useConfig('/config/form.json');
   const userEmail = keycloak.idTokenParsed.email; 
   const userName = keycloak.idTokenParsed.given_name;
-  
+  const userToken = keycloak.token;
+  const [ segments, setSegments ] = useState(null);
+
   const getformID = ( actionLink ) => {
     return actionLink.charAt(actionLink.length-1)
   }
 
+  useEffect(() => {
+    const fetchSegments = async () => {
+      const axiosConfig = {
+
+      }
+      if (config) {
+        const result = await axios.get(`${config.backendURL}segments`, {
```

Comment:
need to catch any errors so that your app doesnt break here. Incase axios throws 5xx or 4xx codes